Dapper/SqlMapper.cs

```diff
@@ -2463,13 +2481,17 @@ private static List<IMemberMap> GetValueTupleMembers(Type type, string[] names)
                         }
                         if (ok)
                         {
-                            Array.Sort(positions, propsArr);
-                            props = propsArr;
+                            props = propsList.ToArray();
+                            Array.Sort(positions, (PropertyInfo[]) props);
                         }
                     }
                 }
             }
-            props = props ?? propsArr.OrderBy(x => x.Name);
+            if(props == null)

```

Comment:
![default](https://user-images.githubusercontent.com/922660/29506609-57ce7086-8655-11e7-9b86-99e17046f9be.png)

In current Dapper code. Can you give `.editorconfig` or some formatting guides for Dapper code?